src/user_functions.js

```diff
@@ -19,7 +19,7 @@ const fs = require("fs");
  *
  * @returns {MemoryModel} the produced canvas
  */
-function draw(objects, automation, configuration) {
+function draw(objects, automation, configuration, style) {
```

Comment:
Whenever you add a parameter, the function documentation also needs to be updated